HW1/P2/P2b.py

```diff
@@ -1,3 +1,31 @@
 from P2 import *
+# Your code here
 
+import numpy
+import matplotlib.pyplot as plt
+import random
 # Your code here
+# 2000 x 2000 pixels
+
+sc.setLogLevel('ERROR')
+X = range(2000)
+Y = range(2000)
+random.shuffle(X)
+random.shuffle(Y)
```

Comment:
By shuffling the data, you achieved your goal of randomizing the partitioning. But this was enabled because you knew how the data was structured. You won't always have that luxury. There are other methods to randomize partitioning that are more general (like .partitionBy() where you pass in a hash function or .repartition()) and more "spark-like".